extensions/wikibase/tests/src/org/openrefine/wikibase/operations/PerformWikibaseEditsOperationTest.java

```diff
@@ -77,7 +77,7 @@ public void testGetTagCandidates() {
     @Test
     public void testLoadChange()
             throws Exception {
-        String changeString = "newItems={\"qidMap\":{\"1234\":\"Q789\"}}\n" + "/ec/\n";
+        String changeString = "newItems={\"qidMap\":{\"1234\":{\"returnedId\":\"Q789\",\"label\":\"Q789\"}}}\n" + "/ec/\n";
```

Comment:
We'll need to make sure the old format of such Change objects can still be deserialized correctly, so that operations run with previous versions of OpenRefine can still be done and undone correctly. So there should be two tests: the current one (unchanged) and another one for the new format.

To make this backwards compatibility simpler, I would recommend using two separate maps (one for the entity ids, one for the names) instead of storing pairs of values in maps. It's probably worth updating the Java types accordingly.

Reply:
That would work, but I think that to ensure backwards compatibility, it would be simpler to use **two separate maps**, as I wrote above. This means:
```java
Map<Long, String> reconToEntityId;
Map<Long, String> reconToName;
```

Reply:
Perfect